extensions/wikia/AppPromoLanding/AppPromoLandingController.class.php

```diff
@@ -129,7 +130,10 @@ public function content( $params ){
 					'count' => 1
 				] )->getVal( 'result' );
 
-				$thumbnail = $img[$item['id']][0]['url'];
+				$thumnail = null;
```

Comment:
![MAJOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-major.png) Remove this unused "$thumnail" local variable. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](http://10.8.74.32:9000/coding_rules#rule_key=php%3AS1481)